doc/topics/releases/releasecandidate.rst

```diff
@@ -2,53 +2,103 @@
 
 .. _release-candidate:
 
-===========================================
-Installing/Testing a Salt Release Candidate
-===========================================
+=====================================
+Install/Test a Salt Release Candidate
+=====================================
 
 When it's time for a new feature release of Salt, follow the instructions below to
 install the latest release candidate of Salt, and try all the shiny new
 features! Be sure to report any bugs you find on `Github
 <https://github.com/saltstack/salt/issues/new/>`_.
 
-Installing Using Packages
-=========================
 
+Install using packages
+======================
 Builds for a few platforms are available as part of the RC at https://repo.saltproject.io/salt_rc/.
-The builds should include the latest version of the OS that is currently available.
+The builds should include the latest version of the operating system that is currently available.
 
 .. note::
 
-    Follow the instructions on https://repo.saltproject.io/,
-    but insert ``salt_rc/`` into the URL between the hostname
-    and the remainder of the path.
+   Beginning with the 3005 (Phosphorus) release of Salt, the Salt Project is
+   changing its packaging system to Tiamat. Any new operating systems added in 3005
+   will only have Tiamat packages. The Salt Project will phase out the old Salt
+   package builds for currently supported operating systems by 3007. See
+   `What is Tiamat? <https://docs.saltproject.io/salt/install-guide/en/latest/topics/upgrade-to-tiamat.html#what-is-tiamat>`_
+   for more information.
 
-    For Redhat Python 3
+To install release candidate packages:
+
+Follow the instructions for your operating system on https://repo.saltproject.io/,
+but insert ``salt_rc/`` into the URL between the hostname and the remainder
+of the file path.
 
-    .. code-block:: bash
 
-        baseurl=https://repo.saltproject.io/salt_rc/py3/redhat/$releasever/$basearch/
+Tiamat packages
+---------------
+For RedHat:
 
-    For Ubuntu Python 3 (replace os_version, with ubuntu version. For example 20.04)
+.. code-block:: bash
 
-    .. code-block:: none
+    baseurl=https://repo.saltproject.io/salt_rc/salt/py3/redhat/$releasever/$basearch/
 
-        deb https://repo.saltproject.io/salt_rc/py3/ubuntu/<os_version>/amd64 focal main
+For Ubuntu, replace the ``<os_version>`` variable with number of the Ubuntu
+version and ``<codename>`` for the release codename.
 
-    For Debian Python 3 (replace os_version, with debian version. For example 10)
+.. code-block:: none
 
-    .. code-block:: none
+    deb https://repo.saltproject.io/salt_rc/salt/py3/ubuntu/<os_version>/amd64 <codename> main
 
-        deb https://repo.saltproject.io/salt_rc/py3/debian/<os_version>/amd64 buster main
+For example, for the 22.04 release of Ubuntu, codename Jammy Jellyfish:
```

Comment:
replace codename Jammy Jellyfish:

with
```
codename jammy
```

since the while it is called Jammy Jellyfish, the actual codename to use is 'jammy'